src/Html.php

```diff
@@ -1301,6 +1301,17 @@ public static function includeHeader(
         }
         $tpl_vars['css_files'][] = ['path' => 'css/palettes/' . $theme . '.scss'];
 
+        // Add specific meta tags for plugins
+        if (isset($PLUGIN_HOOKS[Hooks::ADD_HEADER_TAG]) && count($PLUGIN_HOOKS[Hooks::ADD_HEADER_TAG])) {
+            foreach ($PLUGIN_HOOKS[Hooks::ADD_HEADER_TAG] as $plugin => $header_tags) {
+                if (!Plugin::isPluginActive($plugin)) {
+                    continue;
+                }
+
+                $tpl_vars['header_tags'] = $header_tags;
```

Comment:
If multiple plugins are using this hook, each will override the value defined by the preceding plugin. You should merge all the values returned in `$tpl_vars['header_tags']`.

Reply:
Thanks, We updated the related code.